autoload/ale/fix/buf_registry.vim

```diff
@@ -0,0 +1,5 @@
+let s:default_registry['buf-format'] = {
```

Comment:
I'm pretty sure this is wrong - I've only learned vimscript by example, and I think `s:` is wrong here

Reply:
See https://github.com/dense-analysis/ale/blob/master/autoload/ale/fix/registry.vim